h/resources.py

```diff
@@ -101,7 +101,7 @@ def __getitem__(self, client_id):
             client.__parent__ = Root(self.request)
 
             return client
-        except:
+        except:  # noqa: E722
```

Comment:
Out of curiosity (and unrelated to the actual linting, technically), what led to these bare excepts in the several modules that have them?

Reply:
In this particular case, the exact exception that is thrown if a UUID is invalid is somewhat implementation dependent as it comes from the database layer that SQLAlchemy is using.

In the other modules (the WebSocket server, the search client) we have a couple of places where there is a long tail of errors that could potentially be surfaced and either the details don't really matter (when closing the WS server), or we are just interested in the volume but not the details (in the search client).